templates/developer.rackspace.com/_includes/site-header.html

```diff
@@ -2,8 +2,11 @@
     <div class="header-links">
         <div class="header-links-container">
             <div class="phone-chat-links">
-                <span class="text-link">IRC on Freenode <a href="https://webchat.freenode.net/?channels=rackspace">#rackspace</a></span>
-                <span class="text-link developer-forum"><a href="http://community.rackspace.com/developers/default">developer forum</a></span>
+                <ul>
```

Comment:
I see this patch changes from `<span>` to `<ul>`. Have you tested that in Deconst Integrated?

Reply:
@robb-romans 
I have not. I don't have Deconst Integrated installed. 
I can close this PR until I have time to figure that out.

Reply:
I'll then open an issue for the things in the dev site header that don't work anymore.